### ISSUE TITLE
Installer can't find SkyrimSE

### ISSUE THREAD

**Issue by ivakam** [rockerbacon/lutris-skyrimse-installers#241]
I'm on manjaro trying to install MO2 as per the instructions, but the installer can't seem to find the SkyrimSE installation, instead showing the following:

```
Initial process has started with pid 2019757
Start monitoring process.
Searching for Steam in '/home/ivar/.local/share/Steam'
Found Steam
Searching for Steam in '/home/ivar/.var/app/com.valvesoftware.Steam/.local/share/Steam'
Steam not found in this install path
Searching for game in library '/home/ivar/.local/share/Steam'
Found game
/home/ivar/.cache/lutris/installer/mod-organizer-2/utils/find-library-for-appid.sh: line 44: return: can only 'return' from a function or sourced script
Searching for game in library '/home/ivar/.local/share/Steam'
Found game
/home/ivar/.cache/lutris/installer/mod-organizer-2/utils/find-library-for-appid.sh: line 44: return: can only 'return' from a function or sourced script
ERROR: could not find game with APPID '489830'
/home/ivar/.cache/lutris/installer/mod-organizer-2/utils/find-library-for-appid.sh: line 49: return: can only 'return' from a function or sourced script
```
I'm not sure why it seems to be running parts of the script multiple times even after successfully finding an installation. At this point the installer fails to continue, pressing cancel and "forcing" it to continue just results in other errors and the installation is aborted.

Some relevant info:
- SkyrimSE is running proton 5.0-10 with a fresh pfx (pfx can be found with correct appid under compatdata)
- Lutris is version 0.5.8.4

Any ideas?

**Comment by ivakam** [comment]
After some sleuthing I managed to pin the problem down to steam_libraries taking on multiple correct paths which all get echod back to the main installer script, causing issues. I have no idea why this is even a thing, but forcing it to only contain one single correct path solves the issue.

**Comment by milliams** [comment]
I'm hitting this same thing. I've found that it can be fixed by editing the file `~/.cache/lutris/installer/mod-organizer-2/find_library_for_appid/find-library-for-appid.sh` and changing both the `return 0` and `return 1` into `exit 0` and `exit 1`. To make this work, I started the installation of MO2, and when it paused on the Lutris install screen asking me to chose which game I wanted to manage, I then went and edited the file before continuing.

I'm guessing that the script used to be `source`d but it now executed as a script, as evidenced by `return: can only 'return' from a function or sourced script`?
